README.md

```diff
@@ -100,6 +109,102 @@ var client = contentful.createClient({
 
 You can check other options for the client on our reference documentation
 
+## Advanced features
+
+### Link resolution
+
+contentful.js does by default link resolution unless specified otherwise.
```

Comment:
proper grammar would be to have some commas; also the there is no verb in the sentence - "resolution" is a noun.
```
contentful.js does, by default, resolve links unless specified otherwise.
```
or you could just flip it around
```
contentful.js resolves links by default unless specified otherwise.
```